pkgs/development/python-modules/sphinx_rtd_theme/default.nix

```diff
@@ -8,25 +8,26 @@
 
 buildPythonPackage rec {
   pname = "sphinx_rtd_theme";
-  version = "0.4.3";
+  version = "0.5.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "728607e34d60456d736cc7991fd236afb828b21b82f956c5ea75f94c8414040a";
+    sha256 = "19c31qhfiqbm6y7mamglrc2mc7l6n4lasb8jry01lc67l3nqk9pd";
   };
 
   propagatedBuildInputs = [ sphinx ];
 
   checkInputs = [ readthedocs-sphinx-ext pytest ];
+  CI=1;
```

Comment:
Can we export this variable where it is required?

Reply:
I don't understand what you mean. It is exported for the build and required by the build. Are you saying you would prefer to patch the build script?

Reply:
No, it would be nicer if we did something like

```nix
  preBuild = ''
    export CI=1
  '';
```